### DISCUSSION TITLE
Performance stats for tcplogreceiver

### DISCUSSION THREAD

**Discussion by ochauhan** [open-telemetry/opentelemetry-collector-contrib#7907] [Q&A]
I'm planning to use tcplogreceiver, I would like to know the performance stats for the tcplogreceiver if anyone can share it.
Or Do you have any other procedure by using that I can generate the performance stats report?

**Comment by jpkrohling** [comment]
cc @djaglowski

**Comment by djaglowski** [comment]
Hi @ochauhan. We do not currently have performance benchmarks for most components, including this one. What information specifically are you interested in?

**Reply by ochauhan** [reply]
I looking for throughput of tcplogreceiver, like how much data it can process within the time of interval. 
for example 500MB of data in 1 hour on 2 core CPUs (2+ GHz) and 4GB RAM.
